src/main/java/net/datafaker/transformations/XmlTransformer.java

```diff
@@ -55,12 +55,12 @@ public CharSequence generate(Schema<IN, ?> schema, int limit) {
 
     @Override
     public String getStartStream(Schema<IN, ?> schema) {
-        throw new UnsupportedOperationException();
+        return null;
     }
 
     @Override
     public String getEndStream() {
-        throw new UnsupportedOperationException();
+        return null;
```

Comment:
Wouldn't it be better to return empty string?

Reply:
that's where I'm still not sure, empty string also might be ok here

Reply:
In both cases

L58 too

Reply:
Sorry for some reason I was only seeing one changed. (Maybe I looked at a single commit vs the whole change set 🤷‍♂️)

Reply:
no problem, thanks for the feedback